MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -12,75 +12,76 @@ final class MovieQuizViewController: UIViewController {
     
     private var currentQuestionIndex: Int = 0
     private var correctAnswersCounter: Int = 0
-    private let defaults = UserDefaults.standard
-    
-    private let questions: [QuizQuestion] = [
-        QuizQuestion(image: "The Godfather", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
-        QuizQuestion(image: "The Dark Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
-        QuizQuestion(image: "Kill Bill", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
-        QuizQuestion(image: "The Avengers", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
-        QuizQuestion(image: "Deadpool", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
-        QuizQuestion(image: "The Green Knight", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: true),
-        QuizQuestion(image: "Old", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
-        QuizQuestion(image: "The Ice Age Adventures of Buck Wild", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
-        QuizQuestion(image: "Tesla", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false),
-        QuizQuestion(image: "Vivarium", text: "Рейтинг этого фильма больше чем 6?", correctAnswer: false)
-    ]
+    private let questionAmount: Int = 10
+    private var questionFactory: QuestionFactoryProtocol?
+    private var currentQuestion: QuizQuestion?
+    private var alertPresenter: AlertPresenter?
+    private var statisticService: StatisticService?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        makeImageRoundAndCreateBorder()
-        let firstQuestion = questions[currentQuestionIndex]
-        let convertedData = convert(model: firstQuestion)
-        show(quiz: convertedData)
-        
+        questionFactory = QuestionFactory(delegate: self)
+        alertPresenter = AlertPresenter(delegate: self)
+        statisticService = StatisticServiceImplementation()
+        imageView.layer.cornerRadius = 8
+        questionFactory?.requestQuestion()
     }
     
     @IBAction private func noButtonPressed(_ sender: UIButton) {
         disableButtons()
         let userAnswer = false
-        let currentQuestion = questions[currentQuestionIndex]
+        guard let currentQuestion = currentQuestion else {return}
```

Comment:
У фигурных скобок обычно принято со всех сторон иметь небольшое пространство:
```suggestion
        guard let currentQuestion = currentQuestion else { return }
```

Reply:
Просмотрел ссылки со style guide. Постарался по максимуму найти недочетов в оформлении кода в проекте, но, по мере более детального прочтения статей, их, видимо, найдется еще))

Reply:
Ничего, страшного, по-чуть-чуть пока попробуй :)